11ty/definitions/colorblind.md

```diff
@@ -0,0 +1,44 @@
+---
+title: colorblind (race)
+slug: colorblind-racism
+flag:
+  text: 'Medical appropriation'
+  level: avoid
+defined: true
+excerpt: Colorblind, someone with decreased ability to see color or differences in color; colloquially, belief that racism is no longer a problem and that we all have equal opportunities.
+speech: adjective
+alt_words:
+  - ignorant
+  - insensitive
+  - post-racial
+  - race neutral
+reading:
+  - text: 'What you erase when you say “I don’t see color”'
+    href: https://medium.com/@tinu/what-you-erase-when-you-say-i-dont-see-color-73360346afa7
+  - text: '7 Reasons Why "Colorblindness" Contributes to Racism Instead of Solves It'
+    href: https://everydayfeminism.com/2015/02/colorblindness-adds-to-racism/
+---
+
+colorblind (or color-blind), a condition where someone can only see a limited range of colors or is unable to clearly distinguish different colors. Color blindness can be congenital (from birth) or acquired (due to chronic illness, disease, injury, or old age); colloquially, belief that racism is no longer a problem and that we all have equal opportunities.
+
+## Appropriate Usage
+
+Referring the medical condition as described above
+
+## Inappropriate Usage
+
+As a literary metaphor for ignorance of racial injustice
+
+## Issues
+
+Connoting ignorance or racist microagression with a medical disorder implies that actions we can control (bad choices) are the same as actions that cannot necessarily be controlled (colorblindness). It reinforces the discriminatory idea that disability is bad.
+
+## Impact
+
+Using the word colorblind reinforces the idea that blind and/or vision-impaired people are somehow less than and that disability is bad (see [Ableism](/definitions/ableism)).
```

Comment:
```suggestion
Using the word colorblind in a medically appropriative way can reinforce the idea that Blind and/or vision-impaired people are somehow less than and that disability is bad (see [Ableism](/definitions/ableism)).
```

Reply:
I used most of this but used "blind" (lowercase b) because I'm talking about the medical condition not the Blind culture.